CHANGELOG/CHANGELOG-4.1.md

```diff
@@ -0,0 +1,129 @@
+# Release notes for v4.1.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v4.0.0
+
+## Changes by Kind
+
+### Deprecations
+
+- VolumeSnapshot v1beta1 is deprecated and will be removed in a future release. It is recommended for users to upgrade to VolumeSnapshot CRD version v1 as soon as possible. ([#493](https://github.com/kubernetes-csi/external-snapshotter/pull/493), [@xing-yang](https://github.com/xing-yang))
+
+### API Change
+
+- Changes VolumeSnapshot API storage version from v1beta1 to v1; VolumeSnapshot v1beta1 is deprecated and will be removed in a future release. ([#493](https://github.com/kubernetes-csi/external-snapshotter/pull/493), [@xing-yang](https://github.com/xing-yang))
```

Comment:
Do we also want to make a note that any previously invalid v1beta1 objects have to be deleted before upgrading to this version?

Reply:
Sure.  Added in Deprecations section.